WebApplication1/WebApplication1/Controllers/DrinkController.cs

```diff
@@ -36,9 +38,9 @@ public IActionResult List(string category)
             else
             {
                 if (string.Equals("Alcoholic", _category, StringComparison.OrdinalIgnoreCase))
-                    drinks = drinkRepostory.drinks.Where(p => p.Category.CategoryName.Equals("Alcoholic")).OrderBy(p => p.Name);
```

Comment:
* try to not use this way coz all the rows will be coming from the Database then the filter will be applied 
* even if all these proses were done on the database side try to move this logic to the repository coz it's the repo responsibility not the controller